services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_logs.py

```diff
@@ -62,7 +62,13 @@ async def start_log_feching(self, container_name: str) -> None:
     async def stop_log_fetching(self, container_name: str) -> None:
         logger.debug("Stopping logs fetching from container '%s'", container_name)
 
-        task = self._log_processor_tasks.pop(container_name)
+        task: Optional[Task] = self._log_processor_tasks.pop(container_name, None)
+        if task is None:
+            logger.info(
```

Comment:
is that not a warning? this is not supposed to happen right?

Reply:
It actually can happen more that you would expect. Each time we run docker-compose down. This is one of the steps ran before the docker-compose down command. Since the command can fail this is retried, so it is expected to end up in this situation. That's why it is on info